vuepress/docs/v7.0/tutorials/create/mfe/widget-configuration.md

```diff
@@ -240,25 +249,25 @@ npm run build
 }
 ```
    
-9. Set the **`Custom UI`** so it accepts the "name" config parameter. 
-```javascript
-{
+- Set **`Custom UI`** to accept the "name" config parameter 
+```ftl
     <#assign wp=JspTaglibs[ "/aps-core"]>
     <link rel="stylesheet" type="text/css" href="<@wp.resourceURL />my-widget/static/css/main.073c9b0a.css">
     <script nonce="<@wp.cspNonce />" async src="<@wp.resourceURL />my-widget/static/js/main.e6296e83.js" ></script>
     <@wp.currentWidget param="config" configParam="name" var="configName" />
     <my-widget name="${configName}" />
-}
 ```
 ::: tip
 Multiple `<@wp.currentWidget param` tags can be used when a config MFE supports more than one parameter.
 ::: 
 
-10. Test the full setup by adding the widget into an existing page. 
+8. Test the full setup by adding the widget into an existing page
     
-11. Fill out the "name" field and click `Save`. You can update the widget configuration at any point by clicking `Settings` from the widget actions in the Page Designer.
+9. Fill out the "name" field and click `Save`
+
+> Note: You can update the widget configuration at any point by clicking `Settings` from the widget actions in the Page Designer.
```

Comment:
I don't think this should be in a note because it is REALLY important.  I didn't pay attention to it because it was in a note. If this note had been in the previous step, it would have given a clarity about where to update the name.